data/slds/1.1/point_simplepoint_oneline.sld

```diff
@@ -1 +1 @@
-<?xml version="1.0" encoding="UTF-8" standalone="yes"?><StyledLayerDescriptor version="1.1.0" xsi:schemaLocation="http://www.opengis.net/sld StyledLayerDescriptor.xsd" xmlns="http://www.opengis.net/sld" xmlns:ogc="http://www.opengis.net/ogc" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:se="http://www.opengis.net/se"><NamedLayer><se:Name>Simple Point</se:Name><UserStyle><se:Name>Simple Point</se:Name><se:FeatureTypeStyle><se:Rule><se:Name/><se:PointSymbolizer uom="http://www.opengeospatial.org/se/units/pixel"><se:Graphic><se:Mark><se:WellKnownName>circle</se:WellKnownName><se:Fill><se:SvgParameter name="fill">#FF0000</se:SvgParameter><se:SvgParameter name="fill-opacity">0.5</se:SvgParameter></se:Fill><se:Stroke><se:SvgParameter name="stroke">#0000FF</se:SvgParameter><se:SvgParameter name="stroke-opacity">0.7</se:SvgParameter></se:Stroke></se:Mark><se:Size>6</se:Size></se:Graphic></se:PointSymbolizer></se:Rule></se:FeatureTypeStyle></UserStyle></NamedLayer></StyledLayerDescriptor>
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?><StyledLayerDescriptor version="1.1.0" xsi:schemaLocation="http://www.opengis.net/sld StyledLayerDescriptor.xsd" xmlns="http://www.opengis.net/sld" xmlns:ogc="http://www.opengis.net/ogc" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:se="http://www.opengis.net/se"><NamedLayer><se:Name>Simple Point</se:Name><UserStyle><se:Name>Simple Point</se:Name><se:Title>Simple Point</se:Title><se:FeatureTypeStyle><se:Rule><se:PointSymbolizer><se:Graphic><se:Mark><se:WellKnownName>circle</se:WellKnownName><se:Fill><se:SvgParameter name="fill">#FF0000</se:SvgParameter><se:SvgParameter name="fill-opacity">0.5</se:SvgParameter></se:Fill><se:Stroke><se:SvgParameter name="stroke">#0000FF</se:SvgParameter><se:SvgParameter name="stroke-opacity">0.7</se:SvgParameter></se:Stroke></se:Mark><se:Size>6</se:Size></se:Graphic></se:PointSymbolizer></se:Rule></se:FeatureTypeStyle></UserStyle></NamedLayer></StyledLayerDescriptor>
```

Comment:
Why is this file not indented?

Reply:
That is actually the purpose of this file. It's there to test the unformatted output. "point_simplepoint_**oneline**.sld"